src/cluster-client/Cargo.toml

```diff
@@ -14,6 +14,7 @@ anyhow = "1.0.97"
 http = "1.2.0"
 mz-ore = { path = "../ore", features = ["tracing"] }
 mz-proto = { path = "../proto" }
+mz-repr = { path = "../repr" }
```

Comment:
If we don't like the new dependency, we might be able to move `WallclockLagFn` elsewhere. Suggestions welcome!

Reply:
Maybe a `cluster-types` crate? I think it's fine for the moment.